.github/workflows/spanner-emulator-system-tests.yaml

```
@@ -45,7 +45,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer config repositories.local  --json '{"type":"path", "url": "../Core", "options": {"versions": {"google/cloud-core": "1.45"}}}' -d Spanner
+          composer config repositories.local  --json '{"type":"path", "url": "../Core", "options": {"versions": {"google/cloud-core": "1.49"}}}' -d Spanner
```

The configuration here makes sure the tests are run with `google/cloud-core` from our local repo, rather than pulling it from packagist. But we need to provide a version in order for composer to properly resolve all dependencies. So in this case, we are saying "use the directory `../Core` as `google/cloud-core`, and pretend that it's version `1.45`".

We should match this to what we have in [run-package-tests.sh](https://github.com/googleapis/google-cloud-php/blob/main/.github/run-package-tests.sh#L35) and make it `1.100` (to try and make this obvious)
```suggestion
          # ensure composer uses local Core instead of pulling from packagist
          composer config repositories.local  --json '{"type":"path", "url": "../Core", "options": {"versions": {"google/cloud-core": "1.100"}}}' -d Spanner
```